rpcd/playbooks/roles/rpc_post_upgrade/tasks/post-upgrade-neutron-venv.yml

```diff
@@ -0,0 +1,26 @@
+---
+# Copyright 2016, Rackspace US, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# TODO: verify it's OK to check for errors, and that it won't cause
+# this playbook to fail in the case of failures happening under normal circumstances
+
+- name: Find running neutron services not in venv
+  shell: |
+    pgrep -a "neutron" | awk '{print $2,$3}' | grep -vP "{{ neutron_venv_bin }}/\w+[\d\.\d]?\s{1}{{ neutron_venv_bin }}/\w+"
+  register: neutron_output
+  failed_when: "neutron_output.stdout_lines|length != 0"
+
+- name: Display output of neutron_output
+  debug: var=neutron_output
```

Comment:
On a fail this will report a fatal error which provides the neutron_output.stdout_lines:

Example after changing up the regex a bit:
```
fatal: [danko-aio-01_neutron_agents_container-3e92b25e]: FAILED! => {"changed": true, "cmd": "pgrep -a \"neutron\" | awk '{print $2,$3}' | grep -vP \"/openstack/venvs/neutron-r13.1.0rc1/bin/\\w+[\\d\\.\\d]?\\s{1}/openstack/venvs/neutron-r13.1.0rc1/bin/\\w+\"", "delta": "0:00:00.008751", "end": "2017-04-20 18:36:56.538472", "failed": true, "failed_when_result": true, "rc": 0, "start": "2017-04-20 18:36:56.529721", "stderr": "", "stdout": "/openstack/venvs/neutron-r13.1.0rc1/bin/python2.7 /openstack/venvs/neutron-r13.1.0rc1/bin/neutron-lbaasv2-agent", "stdout_lines": ["/openstack/venvs/neutron-r13.1.0rc1/bin/python2.7 /openstack/venvs/neutron-r13.1.0rc1/bin/neutron-lbaasv2-agent"], "warnings": []}
```

Running in debug would display all neutron_output regardless of failure.

I tried a few different ways to store and then output multiple lines of failures but it wasn't much cleaner than viewing the failure output.